packages/apify/src/actor.ts

```diff
@@ -2319,6 +2333,23 @@ export class Actor<Data extends Dictionary = Dictionary> {
         );
     }
 
+    /**
+     * Get time remaining from the Actor run timeout. Returns `undefined` if not on an Apify platform or the current
+     * run was started without a timeout.
+     */
+    private _getRemainingTime(): number | undefined {
+        const env = this.getEnv();
+        if (env.isAtHome === '1' && env.timeoutAt !== null) {
+            return env.timeoutAt.getTime() - Date.now();
+        }
+        log.warning(
+            'Returning `undefined` instead of remaining time. Using `RemainingTime` argument is only possible when ' +
+                'the Actor is running on the Apify platform and when the timeout for the Actor run is set. ' +
+                `${env.isAtHome}, ${env.timeoutAt}`,
```

Comment:
Given the conditional check above, I suppose this will always print `undefined, null`, right? If so, it IMO doesn't bring any value, and I'd just drop it

```suggestion
```